resources/js/Pages/Results.vue

```diff
@@ -309,6 +312,22 @@
             showSubmitConfirmation( item: string ) {
                 this.lastSubmitted = item;
             },
+            hasChanged(entityId: string) {
+                for (var decisionId in this.decisions[entityId]) {
+                    const decision = this.decisions[entityId][decisionId];
+                    if(decision.review_status !== decision.previous_status) {
+                        return true;
+                    }
+                }
+
+                return false;
+            },
+            storePreviousDecisions(item: string) {
+                for (var decisionId in this.decisions[item]) {
```

Comment:
Sorry, I didn't catch this one before... It's better to use a `const` here. In any case `let` or `const` to be clear about scoping.